docs/glossary.rst

```diff
@@ -8,8 +8,21 @@ Glossary
 
    PasteDeploy
       A system for configuration of WSGI web components in declarative
-      ``.ini`` format.  See http://pythonpaste.org/deploy/.
+      ``.ini`` format.  See https://web.archive.org/web/20161029233359/http://pythonpaste.org/deploy/.
 
    asyncore
-      A standard library module for asynchronous communications.  See
-      http://docs.python.org/library/asyncore.html .
+      A Python standard library module for asynchronous communications.  See :mod:`asyncore`.
```

Comment:
It's being deprecated in the stdlib, instead we have vendored it as `wasyncore`.

Reply:
There are lots of instances of `asyncore` but none for `wasyncore` throughout the docs.

How about I add a `.. deprecated::` to this glossary entry, and add one for `wasyncore`?

As far as all the references to `asyncore`, I can try a simple find/replace, do a quick sanity check, then have you review for accuracy. Sound good?

Reply:
Also what will be the next release number of Waitress, 1.2.0?

Reply:
That works for me.